ngi_pipeline/utils/post_analysis.py

```diff
@@ -0,0 +1,26 @@
+
+from ngi_pipeline.log.loggers import minimal_logger
+from ngi_pipeline.utils.classes import with_ngi_config
+from ngi_pipeline.utils.filesystem import execute_command_line, safe_makedir
+
+import os
+
+def run_multiqc(base_path, project_id, project_name):
+
+    project_path=os.path.join(base_path, 'ANALYSIS', project_id)
+    result_path=os.path.join(base_path, 'ANALYSIS', project_id, 'multiqc')
+    safe_makedir(result_path)
+    command=['multiqc', project_path, '-o', result_path, '-i', project_name, '-n', project_name, '-q', '-f']
```

Comment:
maybe should we move the options to the configuration file in order to avoid future pull reqeusts and re-deployment for changes in the parameters?